### PR TITLE
Bump v1.1.0

### PR DESCRIPTION
**Pull request by lucklove**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
 TiUP Changelog
 
+## [1.1.0] 2020.08.28
+
+- Support config monitor components ([#712](https://github.com/pingcap/tiup/pull/712), [@lucklove](https://github.com/lucklove))
```
Comment (lonng) [inline CHANGELOG.md:5]: How about:
```suggestion
- Support specify customized configuration files for monitor components ([#712](https://github.com/pingcap/tiup/pull/712), [@lucklove](https://github.com/lucklove))
```

**Comment by AstroProfundis** [comment]
It's better to group new features and bug fixes together.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+- Set correct `deploy_dir` of monitoring agents when importing ansible deployed clusters ([#704](https://github.com/pingcap/tiup/pull/704), [@AstroProfundis](https://github.com/AstroProfundis))
+- Support debug scp with --natvie-ssh flag ([#689](https://github.com/pingcap/tiup/pull/689), [@lucklove](https://github.com/lucklove))
```
Comment (lonng) [inline CHANGELOG.md:7]: This is a trivial update, I think we can omit it in the release note.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,30 @@
...
+- Support specifying user group or skipping creating a user in the deploy and scale-out stage
+  - to specify the group: https://github.com/pingcap/tiup/blob/master/examples/topology.example.yaml&#35;L7
+  - to skip creating the user: `tiup cluster deploy/scale-out --skip-create-user xxx` ([#678](https://github.com/pingcap/tiup/pull/678), [@lucklove](https://github.com/lucklove))
+- Add `advertise-status-addr` for tiflash to support host name ([#676](https://github.com/pingcap/tiup/pull/676), [@birdstorm](https://github.com/birdstorm))
+- Support rename cluster by the command `tiup cluster rename <old-name> <new-name>`
+  > Grafana stores some data related to cluster name to its grafana.db. The rename action will NOT delete them. So there may be some useless panel need to be deleted manually. ([#671](https://github.com/pingcap/tiup/pull/671), [@lucklove](https://github.com/lucklove))
+- Fix reverse confirmation on prompt ([#670](https://github.com/pingcap/tiup/pull/670), [@AstroProfundis](https://github.com/AstroProfundis))
```
Comment (lonng) [inline CHANGELOG.md:14]: ditto

**Review by lonng — COMMENTED** [review]
Rest LGTM

File: CHANGELOG.md (hunk 2 above)
Comment (lucklove) [reply CHANGELOG.md]: OK

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,36 @@
 TiUP Changelog
 
+## [1.1.0] 2020.08.28
+
+### New Feature
```
Comment (AstroProfundis) [inline CHANGELOG.md:5]: New Features

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,36 @@
...
+
+- [experimental] Support specify customized configuration files for monitor components ([#712](https://github.com/pingcap/tiup/pull/712), [@lucklove](https://github.com/lucklove))
```
Comment (AstroProfundis) [inline CHANGELOG.md:7]: Support specifying

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,36 @@
...
+- Support specifying user group or skipping creating a user in the deploy and scale-out stage ([#678](https://github.com/pingcap/tiup/pull/678), [@lucklove](https://github.com/lucklove))
+  - to specify the group: https://github.com/pingcap/tiup/blob/master/examples/topology.example.yaml&#35;L7
+  - to skip creating the user: `tiup cluster deploy/scale-out --skip-create-user xxx` 
+- [experimental] Support rename cluster by the command `tiup cluster rename <old-name> <new-name>` ([#671](https://github.com/pingcap/tiup/pull/671), [@lucklove](https://github.com/lucklove))
+  > Grafana stores some data related to cluster name to its grafana.db. The rename action will NOT delete them. So there may be some useless panel need to be deleted manually. 
+- Introduce `tiup cluster clean` command ([#644](https://github.com/pingcap/tiup/pull/644), [@lucklove](https://github.com/lucklove)):
```
Comment (AstroProfundis) [inline CHANGELOG.md:13]: Add the [experimental] tag

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,36 @@
...
+  - Cleanup all data in specified cluster: `tiup cluster clean ${cluster-name} --data`
+  - Cleanup all logs in specified cluster: `tiup cluster clean ${cluster-name} --log`
+  - Cleanup all logs and data in specified cluster: `tiup cluster clean ${cluster-name} --all`
+  - Cleanup all logs and data in specified cluster, excepting the prometheus service: `tiup cluster clean ${cluster-name} --all --ignore-role prometheus`
+  - Cleanup all logs and data in specified cluster, expecting the node `172.16.13.11:9000`: `tiup cluster clean ${cluster-name} --all --ignore-node 172.16.13.11:9000`
+  - Cleanup all logs and data in specified cluster, expecting the host `172.16.13.11`: `tiup cluster clean ${cluster-name} --all --ignore-node 172.16.13.12`
+- Support skipping evicting store when there is only 1 tikv ([#662](https://github.com/pingcap/tiup/pull/662), [@lucklove](https://github.com/lucklove))
+- Supporting import clusters with binlog enabled ([#652](https://github.com/pingcap/tiup/pull/652), [@AstroProfundis](https://github.com/AstroProfundis))
```
Comment (AstroProfundis) [inline CHANGELOG.md:21]: Support importing

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,36 @@
...
+- Support yml source format with tiup-dm ([#655](https://github.com/pingcap/tiup/pull/655), [@july2993](https://github.com/july2993))
+- Support detecting port conflict of monitoring agents between different clusters ([#623](https://github.com/pingcap/tiup/pull/623), [@AstroProfundis](https://github.com/AstroProfundis))
+
+### Bugfix
```
Comment (AstroProfundis) [inline CHANGELOG.md:25]: Fixes

**Review by AstroProfundis — APPROVED** [review]

**Pull request merged**